card/lib/card/view/permission.rb

```diff
@@ -0,0 +1,58 @@
+class Card
+  class View
+    # View permissions support view-specific permission handling
+    #
+    # Views can be configured in {Set::Format::AbstractFormat#view view definitions}
+    # with the `perms` directive, eg
+    #
+    #         # only render if user has permission to update card
+    #         view :myview, perms: :update do...
+    module Permission
+      private
+
+      def approve_view
```

Comment:
if you open the diff to the deleted format/permission.rb, you'll see a satisfying reduction of `view` arguments.